src/Symfony/Component/HttpFoundation/AcceptHeader.php

```diff
@@ -44,15 +44,13 @@ public function __construct(array $items)
     /**
      * Builds an AcceptHeader instance from a string.
      *
-     * @param string $headerValue
-     *
      * @return self
      */
-    public static function fromString($headerValue)
+    public static function fromString(?string $headerValue)
```

Comment:
because we were using it ourselves, we have to accept `null` unless we submit a deprecation on branch 4.4